python/hsfs/engine/python.py

```diff
@@ -515,6 +516,26 @@ def show(
             sql_query, feature_store, online_conn, "default", read_options or {}
         ).head(n)
 
+    def read_vector_db(self, feature_group: "hsfs.feature_group.FeatureGroup", n: int =None, dataframe_type: str="default") -> Union[pd.DataFrame, pl.DataFrame, np.ndarray, List[List[Any]]]:
+        dataframe_type = dataframe_type.lower()
+        if not isinstance(dataframe_type, str) or dataframe_type not in [
+            "pandas",
+            "polars",
+            "numpy",
+            "python",
+            "default",
+        ]:
+            raise FeatureStoreException(
+                f'dataframe_type : {dataframe_type} not supported. Possible values are "default", "pandas", "polars", "numpy" or "python"'
```

Comment:
instead of defining valid dataframe types 2 times, you could just create it once and use the list multiple times (later it could be easy to forget to change it in both places if needed)

Reply:
@bubriks refactor the check to a separate method.